autoload/ale/handlers/eslint.vim

```diff
@@ -42,7 +42,18 @@ function! ale#handlers#eslint#GetCommand(buffer) abort
 
     let l:options = ale#Var(a:buffer, 'javascript_eslint_options')
 
-    return ale#node#Executable(a:buffer, l:executable)
+    " ESLint 6 loads plugins/configs/parsers from the project root
+    " By default, the project root is simply the CWD of the running process.
+    " https://github.com/eslint/rfcs/blob/master/designs/2018-simplified-package-loading/README.md
+    " https://github.com/dense-analysis/ale/issues/2787
+    " Identify project root from presence of node_modules dir.
+    " Note: If node_modules not present yet, can't load local deps anyway.
+    let l:modules_dir = ale#path#FindNearestDirectory(a:buffer, 'node_modules')
+    let l:project_dir = !empty(l:modules_dir) ? fnamemodify(l:modules_dir, ':h:h') : ''
```

Comment:
@kevinoid - for better or worse I'm in a monorepo (using Yarn Workspaces) that has the eslint stuff at the repo root but each "package" has its own package.json and node_modules/ as well (for other dependencies, they just don't have eslint). see [my comment here](https://github.com/dense-analysis/ale/issues/3143#issuecomment-652452362)

seems like this line is cd'ing me into the wrong directory. I wonder if we could change the logic to use the modules_dir that has the executable defined as the project dir?